### ISSUE TITLE
Pylint calls an issue with variables with one character (like x or x1) for the camelCase check

### ISSUE THREAD

**Issue by ollie-iterators** [PyCQA/pylint#8118]
### Bug description

Variables with one letter do not pass the camelCase check. This happens when there is only one character in the variable (like x) or if there is a character and a number (like x1)

### Command used

```shell
N/A
```

### Pylint output

```shell
Variable name "x" doesn't conform to camelCase naming stylePylint(C0103:invalid-name)
Variable name "x1" doesn't conform to camelCase naming stylePylint(C0103:invalid-name)
```

### Expected behavior

The variables with one character or a character and a number work for pylint.

### Pylint version

```shell
pylint 2.15.10
astroid 2.13.3
Python 3.11.1 (v3.11.1:a7a450f84a, Dec  6 2022, 15:24:06) [Clang 13.0.0 (clang-1300.0.29.30)]
```

### OS / Environment

macOS 13.2

**Comment by DanielNoord** [comment]
Do you have the code? I think this is a poorly phrased message telling you that the variable should have a length of at least 3.

**Comment by Pierre-Sassoulas** [comment]
Duplicate of https://github.com/PyCQA/pylint/issues/2018 and https://github.com/PyCQA/pylint/issues/7305
